box2dbody.cpp

```diff
@@ -243,6 +244,26 @@ Box2DFixture *Box2DBody::at_fixture(QQmlListProperty<Box2DFixture> *list, int in
     return body->mFixtures.at(index);
 }
 
+QPointF Box2DBody::originOffset(bool bodyToTarget)
+{
+    Q_ASSERT(mTarget);
+
+    QPointF origin = -mTarget->transformOriginPoint();
+    qreal c = qCos(-mBodyDef.angle);
+    qreal s = qSin(-mBodyDef.angle);
+
+    if(bodyToTarget) {
```

Comment:
Yes, I also worried about this parameter since it's unintuitive in this context. I just wanted to avoid code duplication here.